pex/tools/commands/venv.py

```diff
@@ -164,6 +164,16 @@ def add_arguments(cls, parser):
                 )
             ),
         )
+        parser.add_argument(
+            "--non-hermetic",
```

Comment:
So, the name says more than it does. The venv pex is still hermetic, its only the console scripts that are not. I am personally unafraid of long names - blame Java? - and would be super happy with `--non-hermetic-scripts`.

Reply:
👍 will rename